### ISSUE TITLE
handleOpenURL not triggering on Android?

### ISSUE THREAD

**Issue by jedda23** [EddyVerbruggen/Custom-URL-scheme#69]
Hi. I apologize if this has been addressed but I am not seeing it anywhere. I have banging my head on getting this plugin working and probably just missing something. The basic issue I am having is getting the handleOpenURL to trigger on Android. It works fine on iOS. I've boiled my test app down to the bare minimum to remove any other variables.  Although I can get my app to open from url on android, I cannot get handleOpenURL to run on Android. This is on Phonegap Build

My basic index.html page is as follows
```
    <!DOCTYPE HTML>
    <html>
    <head>
	<script>
		function handleOpenURL(url) {
		  setTimeout(function() {
			alert("received url: " + url);
		  }, 0);
		  
		}
	</script>
    </head>
    </html>
```
in my config I have the following:
```
 <gap:plugin name="nl.x-services.plugins.launchmyapp" >
    <param name="URL_SCHEME" value="oeflex" />
</gap:plugin>
```
Thanks in advance for any assistance you can provide. I am hoping this is just a face palm...

**Comment by EddyVerbruggen** [comment]
I think PhoneGap Build has an older version of the plugin - which one are you using?

**Comment by jedda23** [comment]
3.2

**Comment by EddyVerbruggen** [comment]
I've tweaked just that for Android in 3.2.1 and 3.2.2 but that's not available at PGB. However, to assist you I've just published it to the cordova plugin registry: http://plugins.cordova.io/#/package/nl.x-services.plugins.launchmyapp

If you update your `config.xml` to this you'll have the 3.2.2 version:

```xml
<gap:plugin name="nl.x-services.plugins.launchmyapp" version="3.2.2" source="plugins.cordova.io">
  <param name="URL_SCHEME" value="oeflex" />
</gap:plugin>
```

Let me know if this solves your problem. Certainly not a face palm ;)

**Comment by jedda23** [comment]
Hmm. 3.2.2 pulled in but seemingly still the same issue. Have to run for a bit but will look closer in a few hours.

**Comment by jedda23** [comment]
Unfortunately this doesn't seem to be helping. Still works fine in iOS but nothing in Android with the 3.2.2 version.

**Comment by EddyVerbruggen** [comment]
Then it must be something else. Can you share your entire project with me? I'm guessing something is wrong with loading cordova.js.

**Comment by jedda23** [comment]
Ok got it working. Your comment made me a see a dumb mistake in my simplified test package. I had ommited the ```<script src="phonegap.js"></script>``` in the index.html file. I guess I didn't notice because it continued to work in iOS... However the root issue was indeed that it needed the 3.2.2 version which I just now confirmed works perfectly in my full project file as well.

Thanks so much for you assistance.
